dangerfile.js

```diff
@@ -120,16 +124,30 @@ fetch(commitURL(parentOfOldestCommit)).then(async response => {
         'ENV',
       ];
 
-      const mdRows = changedFiles.map(r => [
-        r.filename,
-        emojiPercent(r.prevFileSizeChange),
-        emojiPercent(r.prevGzipSizeChange),
-        r.prevSize,
-        r.prevFileSize,
-        r.prevGzip,
-        r.prevGzipSize,
-        r.bundleType,
-      ]);
+      const mdRows = changedFiles.map(
+        r =>
+          r.bundleType.includes('PROD')
+            ? boldRow([
+                r.filename,
+                emojiPercent(r.prevFileSizeChange),
```

Comment:
We could change signature of this to `addPercent(text, includeEmoji)`. Because we want percent sign in both cases. This lets us make the call unconditional, with different boolean values depending on the branch. Similarly, we could have `boldRow` accept a boolean.

Then you could do

```js
const isProd = r.bundleType.includes('PROD');
return setBoldness([
  r.filename,
  addPercent(r.prevFileSizeChange, isProd),
  // ...
], isProd);
```

and not duplicate the column list twice.

Reply:
Thanks. I have revised the codes per your comments.